wfe/wfe_test.go

```diff
@@ -1818,3 +1822,37 @@ func TestDeactivateAuthorization(t *testing.T) {
 		  ]
 		}`)
 }
+
+func TestDeactivateRegistration(t *testing.T) {
```

Comment:
Can you add a test for the case where the `wfe` returns a err for trying to deactivate a non-valid registration?